src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,11 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    /**
+     * Checks that command can modify data by command type
```

Comment:
Good job on remembering to add header comments for this method. 
However, do take note that it should be written for all your polymorphic methods so that users will know how this method behaves exactly. Also, you should add a `@return` to specify what is being returned by this method.